src/odemis/gui/util/img.py

```diff
@@ -929,7 +929,8 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
                                orientation)
         # Round "almost 0" (due to floating point errors) to 0
         if abs(tick_value) < epsilon:
-            tick_value = 0
+            # tick_value = 0
```

Comment:
Why is that needed? It should eventually go back. Maybe you need to tweak the computation of `epsilon`?